<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>846</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="widget_8" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>登录</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <widget class="QLabel" name="label_2">
               <property name="geometry">
                <rect>
                 <x>9</x>
                 <y>0</y>
                 <width>91</width>
                 <height>39</height>
                </rect>
               </property>
               <property name="text">
                <string>数据库地址:</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="Sql_Address">
               <property name="geometry">
                <rect>
                 <x>100</x>
                 <y>10</y>
                 <width>151</width>
                 <height>31</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget" native="true">
              <widget class="QLabel" name="label">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>0</y>
                 <width>83</width>
                 <height>41</height>
                </rect>
               </property>
               <property name="text">
                <string>数据库名:</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="Sql_Name">
               <property name="geometry">
                <rect>
                 <x>100</x>
                 <y>10</y>
                 <width>141</width>
                 <height>31</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_4" native="true">
              <widget class="QLineEdit" name="Sql_Table_Name">
               <property name="geometry">
                <rect>
                 <x>100</x>
                 <y>10</y>
                 <width>141</width>
                 <height>31</height>
                </rect>
               </property>
              </widget>
              <widget class="QLabel" name="label_3">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>10</y>
                 <width>72</width>
                 <height>15</height>
                </rect>
               </property>
               <property name="text">
                <string>表名：</string>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_5" native="true">
              <widget class="QLineEdit" name="User_Name">
               <property name="geometry">
                <rect>
                 <x>90</x>
                 <y>0</y>
                 <width>151</width>
                 <height>31</height>
                </rect>
               </property>
              </widget>
              <widget class="QLabel" name="label_4">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>10</y>
                 <width>72</width>
                 <height>15</height>
                </rect>
               </property>
               <property name="text">
                <string>用户名：</string>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_6" native="true">
              <widget class="QLineEdit" name="User_Password">
               <property name="geometry">
                <rect>
                 <x>90</x>
                 <y>10</y>
                 <width>141</width>
                 <height>31</height>
                </rect>
               </property>
              </widget>
              <widget class="QLabel" name="label_5">
               <property name="geometry">
                <rect>
                 <x>40</x>
                 <y>20</y>
                 <width>72</width>
                 <height>15</height>
                </rect>
               </property>
               <property name="text">
                <string>密码:</string>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>功能区</string>
           </property>
           <widget class="QWidget" name="widget_9" native="true">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>100</y>
              <width>161</width>
              <height>171</height>
             </rect>
            </property>
            <widget class="QPushButton" name="Button_Login">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>151</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string>登入</string>
             </property>
            </widget>
            <widget class="QPushButton" name="Button_Exit">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>70</y>
               <width>151</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string>退出</string>
             </property>
            </widget>
            <widget class="QPushButton" name="Button_Error_List">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>120</y>
               <width>151</width>
               <height>51</height>
              </rect>
             </property>
             <property name="text">
              <string>错题集</string>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>作答区</string>
        </property>
        <widget class="QWidget" name="widget_7" native="true">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>390</y>
           <width>591</width>
           <height>91</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QRadioButton" name="Button_A">
            <property name="text">
             <string>A</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="Button_B">
            <property name="text">
             <string>B</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="Button_C">
            <property name="text">
             <string>C</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="Button_D">
            <property name="text">
             <string>D</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="widget_10" native="true">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>611</width>
           <height>361</height>
          </rect>
         </property>
         <widget class="QTextEdit" name="Show_Questions">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>601</width>
            <height>361</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="widget_11" native="true">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>490</y>
           <width>571</width>
           <height>131</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="Button_Last">
            <property name="text">
             <string>上一题</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="Button_Next">
            <property name="text">
             <string>下一题</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="Button_Answer">
            <property name="text">
             <string>查看答案</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="Button_Again_All">
            <property name="text">
             <string>全部重做</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="widget_12" native="true">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>630</y>
           <width>601</width>
           <height>151</height>
          </rect>
         </property>
         <widget class="QGroupBox" name="Answer">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>591</width>
            <height>141</height>
           </rect>
          </property>
          <property name="title">
           <string>答案：</string>
          </property>
          <widget class="QTextEdit" name="Show_Answer">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>581</width>
             <height>111</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Button_Exit</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>618</y>
    </hint>
    <hint type="destinationlabel">
     <x>1309</x>
     <y>589</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
